ReactNativePermissions.js

```diff
@@ -17,6 +17,7 @@ const RNPTypes = {
 		'notification',
 		'backgroundRefresh',
 		'speechRecognition',
+        'mediaLibrary',
```

Comment:
An upvote for this! I want to be able to download mp3 files on iOS devices. For Android I have the get permissions use case working... For iOS I can into the 'storage' not defined error and with some Googling I got here at this pull request.
Is it also possible to make the latest version available on NPM?
Thanks!